### DISCUSSION TITLE
Migration from MariaDB to PostgreSQL

### DISCUSSION THREAD

**Discussion by gruesomehit** [requarks/wiki#4043] [Help / Questions] [answered]
I would like to migrate from MariaDB 10.5 to PostgreSQL because I want to use the Advanced Search Engine. So, would be complicated to migrate between databases? Can I use my last backup from the Local File System section on a fresh PostgreSQL installation?

Thanks in advance.

Docker Wikijs 2.5.201, MariaDB 1.5.10

**Comment by NGPixel** [comment, marked as answer]
Yes, this is the recommended way of migrating. Either use the local file or git storage to move over your pages into a new postgres installation.

**Reply by traeblain** [reply]
This keeps the pages data, but what about the settings?  Mail/Theme/Storage/Security/etc?  Would like to migrate now, but all these items are only in the mysql database.

**Reply by NGPixel** [reply]
These cannot be migrated at the moment.
